feature/gnoi/system/tests/chassis_reboot_status_and_cancel_test/chassis_reboot_status_and_cancel_test.go

```diff
@@ -206,3 +188,16 @@ func TestCancelReboot(t *testing.T) {
 		t.Errorf("rebootStatus.GetActive(): got %v, want false", rebootStatus.GetActive())
 	}
 }
+
+func getSubCompPath(t *testing.T, dut *ondatra.DUTDevice) *tpb.Path {
+	t.Helper()
+	controllerCards := components.FindComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD)
+	if len(controllerCards) == 0 {
+		t.Fatal("No controller card components found in DUT.")
+	}
+	activeRP := controllerCards[0]
+	if len(controllerCards) == 2 {
```

Comment:
Would there be a possible scenario where controllerCards is greater than 2?

Reply:
I am not aware of a device with more than two rps, so the length should be 1 or two.

Reply:
+1, we will have devices with 2 control modules only